src/views/Zap/ZapStakeAction.tsx

```diff
@@ -281,7 +281,7 @@ const ZapStakeAction: React.FC = () => {
   return (
     <Root>
       <ZapStakeHeader images={inputTokenImages} />
-      <Typography>
+      <Typography fontSize="15px" lineHeight="24px" mb="3px">
         <Trans>You Pay</Trans>
```

Comment:
Also, just as a question, can we do anything to standardize these font sizes?

Reply:
Yeah, we need to basically remap typography styles from the new design system.
I started to go down the path, but it started to impact lots of things, and was making assumptions that everything was using typography tags (they're not). 
We can leverage Typography tags and 'variants' to map to this: https://www.figma.com/file/nmW0n5E0O2Nxo3Q0RRLQYo/Tokens-V3?node-id=38%3A4873

But I need to sync back up with tutti so we can agree on which sizes map to certain things.. ie 15px = default. 28px = h1, etc etc.. 

It was a bit faster to manually adjust these one off cases for now in isolation, and circle back. ill add a ticket so we're tracking it